test/cql-pytest/test_tablets.py

```diff
@@ -45,3 +45,10 @@ def test_create_loop_with_tablets(cql, test_keyspace_128_tablets):
     for i in range(100):
         cql.execute(f"CREATE TABLE {table} (p int PRIMARY KEY, v int)")
         cql.execute("DROP TABLE " + table)
+
+
+def test_alter_cannot_change_vnodes_to_tablets(cql, skip_without_tablets):
```

Comment:
I would add a comment explaining that **currently** we don't allow ALTERing vnodes to tablets. I doubt we can keep this forbidden forever. @tgrabiec do we  have an issue about supporting ALTER from vnodes to tablets in the future - or there is a reason why we will never support it?